Panel.c

```diff
@@ -503,5 +506,46 @@ int Panel_getCh(Panel* this) {
 #ifdef HAVE_SET_ESCDELAY
    set_escdelay(25);
 #endif
-   return getch();
+
+   int ch;
+#ifdef HAVE_LIBNCURSESW
+   static char wchbuf[MB_LEN_MAX];
+   static int wchbuf_len = 0, wchbuf_i = 0;
+
+   int wchrv;
+   mbstate_t wst;
+   wint_t wkey;
+   wchar_t wch;
+
+   if (wchbuf_i < wchbuf_len) {
+      ch = wchbuf[wchbuf_i++];
+   } else {
+      last_wkey = 0;
+      wchrv = get_wch(&wkey);
+      wch = (wchar_t)wkey;
+      if (wchrv == OK) {
+         last_wkey = wkey;
+         memset(&wst, 0, sizeof(wst));
+         wchbuf_i = 0;
+         wchbuf_len = (int)wcrtomb(wchbuf, wch, &wst);
+         if (wchbuf_len < 0) {
```

Comment:
See return value for `wcrtomb` at https://en.cppreference.com/w/c/string/multibyte/wcrtomb; should check for `(size_t)-1`.